migrations/advanced_permissions_phase_2.go

```diff
@@ -71,17 +71,17 @@ func (worker *Worker) runAdvancedPermissionsPhase2Migration(lastDone string) (bo
 
 	if progress.CurrentTable == "TeamMembers" {
 		// Run a TeamMembers migration batch.
-		if result := <-worker.app.Srv.Store.Team().MigrateTeamMembers(progress.LastTeamId, progress.LastUserId); result.Err != nil {
-			return false, progress.ToJson(), result.Err
+		if result, err := worker.app.Srv.Store.Team().MigrateTeamMembers(progress.LastTeamId, progress.LastUserId); err != nil {
+			return false, progress.ToJson(), err
 		} else {
-			if result.Data == nil {
+			if result == nil {
 				// We haven't progressed. That means that we've reached the end of this stage of the migration, and should now advance to the next stage.
 				progress.LastUserId = strings.Repeat("0", 26)
 				progress.CurrentTable = "ChannelMembers"
 				return false, progress.ToJson(), nil
 			}
 
-			data := result.Data.(map[string]string)
+			data := result
```

Comment:
probably this copy is not needed, you can use `result` instead of `data` everywhere or use `data` instead of `result` everywhere.

Reply:
@jespino alright let me make the changes.